### PR TITLE
Remove the rules_python_external README

### PR DESCRIPTION
**Pull request by thundergolfer**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Does not include precompiled binaries, eg. `.par` files. See [CONTRIBUTING.md](/CONTRIBUTING.md) for info
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Description

This `README` was carried over during the merge of `rules_python_external`, and it has info that's duplicating and sometimes contradictory of the documentation in the root `README`. I'd favour just removing it. 

If there's anything in this `README` that you think should remain in the repo, I can pull it out and put it in the root `README` under _"Using the packaging rules"_.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: python/pip_install/README.md [hunk 1]
```diff
@@ -1,134 +0,0 @@
...
-)
-
-# If you need to depend on the wheel dists themselves, for instance to pass them
-# to some other packaging tool, you can get a handle to them with the whl_requirement macro.
-filegroup(
-    name = "whl_files",
-    data = [
-        whl_requirement("boto3"),
-    ]
-)
```
Comment (hrfuller) [inline python/pip_install/README.md:84]: Might be good to document this in the packaging rules.

File: python/pip_install/README.md [hunk 2]
```diff
@@ -1,134 +0,0 @@
...
-pip_install(
-    name = "py_deps",
-    requirements = "//:requirements.txt",
-    # (Optional) You can provide a python interpreter (by path):
-    python_interpreter = "/usr/bin/python3.8",
-    # (Optional) Alternatively you can provide an in-build python interpreter, that is available as a Bazel target.
-    # This overrides `python_interpreter`.
-    # Note: You need to set up the interpreter target beforehand (not shown here). Please see the `example` folder for further details.
-    #python_interpreter_target = "@python_interpreter//:python_bin",
-)
```
Comment (hrfuller) [inline python/pip_install/README.md:61]: This examples still seems useful to document the `python_interpreter[_target]` arguments.

**Review by hrfuller — APPROVED** [review]
LGTM. Up to your discretion to add the chunks I commented on to the top level README

File: python/pip_install/README.md (hunk 2 above)
Comment (thundergolfer) [reply python/pip_install/README.md:61]: Agreed. It documented in the `README` but in words not code examples.

File: python/pip_install/README.md (hunk 1 above)
Comment (thundergolfer) [reply python/pip_install/README.md:84]: I'll add a sub-section for it 👍

**Pull request merged**